proto/rill/runtime/v1/queries.proto

```diff
@@ -173,6 +180,58 @@ message MetricsViewToplistResponse {
   repeated google.protobuf.Struct data = 2;
 }
 
+// Request message for QueryService.MetricsViewComparisonToplist
+message MetricsViewComparisonToplistRequest {
+  string instance_id = 1;
+  string metrics_view_name = 2;
+  string dimension_name = 3;
+  repeated string measure_names = 4;
+  TimeRange base_time_range = 5;
+  TimeRange comparison_time_range = 6;
+  repeated MetricsViewComparisonSort sort = 7;
+  MetricsViewFilter filter = 8;
+  int64 limit = 9;
+  int64 offset = 10;
+  int32 priority = 11;
+}
+
+// Response message for QueryService.MetricsViewComparisonToplist
+message MetricsViewComparisonToplistResponse {
+  repeated MetricsViewComparisonRow data = 1;
+}
+
+message TimeRange {
+  google.protobuf.Timestamp start = 1;
+  google.protobuf.Timestamp end = 2;
+}
+
+enum ComparisonSortType {
+  COMPARISON_SORT_TYPE_UNSPECIFIED = 0;
+  COMPARISON_SORT_TYPE_COMPARISON_VALUE = 1;
+  COMPARISON_SORT_TYPE_BASE_VALUE = 2;
+  COMPARISON_SORT_TYPE_DELTA = 3;
```

Comment:
1. nit: put base value before comparison value
2. Should support both `delta_rel` and `delta_abs`?

Reply:
2. Ordering of abs delta and relative delta is always the same